wordpress-20-04/files/opt/digitalocean/wp_setup.sh

```diff
@@ -106,4 +106,25 @@ wp plugin install wp-fail2ban --allow-root --path="/var/www/html"
 wp plugin activate wp-fail2ban --allow-root --path="/var/www/html"
 chown -Rf www-data.www-data /var/www/
 cp /etc/skel/.bashrc /root
+
+if [ -f "/root/.digitalocean_dbaas_password" ]; then
+  echo "DBaaS credentials found, applying..."
+  # grab all the data from the password file
+  username=$(sed -n "s/^mysql_username=\"\(.*\)\"$/\1/p" /root/.digitalocean_dbaas_password)
+  password=$(sed -n "s/^mysql_password=\"\(.*\)\"$/\1/p" /root/.digitalocean_dbaas_password)
+  host=$(sed -n "s/^mysql_host=\"\(.*\)\"$/\1/p" /root/.digitalocean_dbaas_password)
+  port=$(sed -n "s/^mysql_port=\"\(.*\)\"$/\1/p" /root/.digitalocean_dbaas_password)
+  database=$(sed -n "s/^mysql_database=\"\(.*\)\"$/\1/p" /root/.digitalocean_dbaas_password)
+
+  # update the wp-config.php with stored credentials
+  sed -i "s/'DB_USER', '.*'/'DB_USER', '$username'/g" /var/www/html/wp-config.php;
+  sed -i "s/'DB_NAME', '.*'/'DB_NAME', '$database'/g" /var/www/html/wp-config.php;
+  sed -i "s/'DB_PASSWORD', '.*'/'DB_PASSWORD', '$password'/g" /var/www/html/wp-config.php;
+  sed -i "s/'DB_HOST', '.*'/'DB_HOST', '$host:$port'/g" /var/www/html/wp-config.php;
+
```

Comment:
I totally forgot about this, but we need to add this as well:
```
/** Connect to MySQL cluster over SSL **/
define( 'MYSQL_CLIENT_FLAGS', MYSQLI_CLIENT_SSL );
```
word for word will always be the same.

Reply:
ahh thanks for that, I did not know!

Reply:
for sure, I just peeked at the instructions I wrote to double check

Reply:
added!